packages/cozy-sharing/src/SharingBanner/locales/en.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
We extracted the localization from cozy-drive. Problem is that we do not have the French translation which is managed by Transifex in cozy-drive. Should we use Transifex here, or should we dump a fr.json?

Reply:
No. You should put this content in the already existing files https://github.com/cozy/cozy-libs/tree/master/packages/cozy-sharing/locales

Reply:
Yes, I hesitated but decided to follow the "everything in one directory". I guess it goes too far when it's about localization